iModelCore/ECDb/ECDb/SchemaWriter.cpp

```diff
@@ -2375,7 +2376,7 @@ BentleyStatus SchemaWriter::UpdateProperty(Context& ctx, PropertyChange& propert
             if (ctx.IgnoreIllegalDeletionsAndModifications())
                 {
                 ctx.Issues().ReportV(IssueSeverity::Info, IssueCategory::BusinessProperties, IssueType::ECDbIssue, ECDbIssueId::ECDb_0343,
-                    "Changes to ECProperty %s.%s skipped because %s a property is not supported without an appropriate 'AllowUnitChange' custom attribute applied the property.",
+                    "Ignoring upgrade error: Changes to ECProperty %s.%s skipped because %s a property is not supported without an appropriate 'AllowUnitChange' custom attribute applied the property. Error suppressed as DoNotFailForDeletionsOrModifications flag is set.",

```

Comment:
```suggestion
                    "Changes to ECProperty %s.%s not allowed because %s a property is not supported without an appropriate 'AllowUnitChange' custom attribute applied the property. Error suppressed, property will not be changed.",

```